File: java/src/main/java/com/experian/dvs/client/address/format/components/AdditionalElements.java
package com.experian.dvs.client.address.format.components;

import com.experian.dvs.client.server.address.format.RestApiAddressFormatComponents;

public class AdditionalElements {
    private final AdditionalLocality locality;

    // Constructor
    public AdditionalElements(RestApiAddressFormatComponents.RestApiAddressComponentAdditionalElements locality) {
        this.locality = locality.getLocality() != null ? new AdditionalLocality(locality.getLocality()) : null;
    }

    // Getters
    public AdditionalLocality getLocality() {
        return locality;
    }

}

File: csharp/DVSClient/DVSClient/address/format/components/AdditionalElements.cs
using DVSClient.Server.Address.Format;

namespace DVSClient.Address.Format.Components
{
    public class AdditionalElements
    {
        public AdditionalLocality? Locality { get; }

        public AdditionalElements(RestApiAddressFormatComponents.RestApiAddressComponentAdditionalElements api)
        {
            Locality = api.Locality != null ? new AdditionalLocality(api.Locality) : null;
        }
    }
}